src/diem/testing/suites/conftest.py

```diff
@@ -48,7 +49,9 @@ def stub_wallet_app(start_stub_wallet: Tuple[AppConfig, App]) -> App:
 
 @pytest.fixture(scope="package")
 def start_stub_wallet(diem_client: jsonrpc.Client) -> Tuple[AppConfig, App]:
-    conf = AppConfig(name="stub-wallet", server_conf=ServerConfig(**dmw_stub_server()))
+    conf = AppConfig(
+        name="stubwallet", server_conf=ServerConfig(**dmw_stub_server()), diem_id_domain=dmw_stub_diem_id_domain()
```

Comment:
I think we probably should keep original name "stub-wallet", don't see a reason to rename it.